back-office/advanced-parameters/administration/administration.md

```diff
@@ -0,0 +1,16 @@
+# **SPECIFICATIONS - ADMINISTRATION PAGE**
+
+
+## General
+
+_As a user, I want to be able to turn my online store into a products catalog._
```

Comment:
If we are talking about the development environment only, it doesn't concern "online stores"

Reply:
Oops, it probably comes from a wrong copy-paste action. It is not related at all, I'll delete this US, thanks for catching it!